### PR TITLE
LIVE-1388: remove tezos from experimental

### PR DESCRIPTION
**Pull request by henri-ly**
## Context (issues, jira)

https://ledgerhq.atlassian.net/browse/LIVE-1388

## Description / Usage

No more XTZ libcore 🤞 

## Expectations

- [ ] **Test coverage: The changes of this PR are covered by test.** Unit test were added with mocks when depending on a backend/device.
- [ ] **No impact: The changes of this PR have ZERO impact on the userland.** Meaning, we can use these changes without modifying LLD/LLM at all. It will be a "noop" and the maintainers will be able to bump it without changing anything.

### CODE REVIEW

File: src/account/support.ts [hunk 1]
```diff
@@ -20,20 +20,6 @@ import { getMainAccount } from "../account";
 import { getAccountBridge } from "../bridge";
 import jsBridges from "../generated/bridge/js";
 
-const experimentalIntegrations = ["tezos"];
```
Comment (haammar-ledger) [inline src/account/support.ts:23]: Please just remove tezos, we still need that for incoming Cosmos-JS :pray:

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/account/support.ts (hunk 1 above)
Comment (henrily-ledger) [reply src/account/support.ts:23]: Oups there was an env
sorry, was just trigerred by ts i'll find a work around

**Review by ghost — APPROVED** [review]
:rocket:

**Review by juan-cortes — APPROVED** [review]

**Comment by gre** [comment]
we need one rollout of "experimental" first to happen before merging.

**Pull request merged**